src/transformers/modeling_ctrl.py

```diff
@@ -330,8 +330,10 @@ def forward(
         elif input_ids is not None:
             input_shape = input_ids.size()
             input_ids = input_ids.view(-1, input_shape[-1])
```

Comment:
The problem is this line here:
This changes the shape of `input_ids` and therefore also the `batch_size`